bad checksum for ocaml-migrate-parsetree

I use opam in a docker container to install some packages but I recently got this error in my CI:

[topkg.1.0.1] downloaded from http://erratique.ch/software/topkg/releases/topkg-1.0.1.tbz
[ERROR] The sources of the following couldn't be obtained, aborting:
          - ocaml-migrate-parsetree.1.4.0: Bad checksum

Anyway I can fix that?

Here is the line in the Dockerfile:
Step 6/18 : RUN git pull && opam update && opam switch 4.07 &&   opam install dune yaml &&   opam install pfff pfff.0.39.3

I run this in the docker image FROM ocaml/opam2:alpine

--- a/packages/ocaml-migrate-parsetree/ocaml-migrate-parsetree.1.4.0/opam
+++ b/packages/ocaml-migrate-parsetree/ocaml-migrate-parsetree.1.4.0/opam
@@ -31,7 +31,7 @@ url {
   src:
     "https://github.com/ocaml-ppx/ocaml-migrate-parsetree/releases/download/v1.4.0/ocaml-migrate-parsetree-v1.4.0.tbz"
   checksum: [
-    "sha256=231fbdc205187b3ee266b535d9cfe44b599067b2f6e97883c782ea7bb577d3b8"
-    "sha512=61ee91d2d146cc2d2ff2d5dc4ef5dea4dc4d3c8dbd8b4c9586d64b6ad7302327ab35547aa0a5b0103c3f07b66b13d416a1bee6d4d117293cd3cabe44113ec6d4"
+    "sha256=dcd77cd7090ce181a87df08a910ab2935da466f37db8e674e5a966d49743de64"
+    "sha512=659d6429a6817a867eab28a1fac03a47fdcd9cf8c0861046580013406dec98a8380405785312c3ebb4dc115250aba70c17ce275813083a213dd6577105ea0ba2"
   ]
 }

--- a/packages/ocaml-migrate-parsetree/ocaml-migrate-parsetree.1.4.0/opam
+++ b/packages/ocaml-migrate-parsetree/ocaml-migrate-parsetree.1.4.0/opam
@@ -17,7 +17,7 @@ depends: [
   "result"
   "ppx_derivers"
   "dune" {>= "1.9.0"}
-  "ocaml" {>= "4.02.3"}
+  "ocaml" {>= "4.08.0"}
 ]
 synopsis: "Convert OCaml parsetrees between different versions"
 description: """

